FirebaseAppCheck/Sources/AppAttestProvider/Errors/FIRAppAttestRejectionError.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRAppAttestRejectionError : NSError
+
+- (instancetype)init;
```

Comment:
The App Attest provider is already pretty big (close to be bigger than the rest of App Check SDK), so it seems it will make sense to separate it in the future. This error is specific for App Attest, so having it independent from Core will make separation easier in the future. Also FIRAppCheckErrorUtil was intended to be used mostly for errors without specific classes.

Reply:
SGTM, thanks